test/Microsoft.AspNet.Mvc.TestCommon/TestFileChangeToken.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Extensions.Primitives
 {
-    internal class TestFileChangeToken : IChangeToken
+    public class TestFileChangeToken : IChangeToken
```

Comment:
This is a test _shared source_ project. Should be no need for any access beyond `internal`.